src/checkbox-table/checkbox-table.tsx

```diff
@@ -34,7 +33,8 @@ interface Props {
     onChange: (data: CheckboxTableData) => void;
     selectAllLabel: string;
     isDisabledCell?: (columnKey: string, rowKey: string) => boolean;
-    selectAllColumnLabel: string;
+    /** @deprecated */
+    selectAllColumnLabel?: string;
```

Comment:
Since we already have some breaking changes (even though it's just regarding styles) i'd prefer to remove it.